ledger/daml-on-sql/README.rst

```diff
@@ -0,0 +1,742 @@
+--------------------------------------------
+
+A database metric. Time spent removing deduplication information after the failure of a
+command. Deduplication information is persisted to ensure the continued
+working of the deduplication mechanism across restarts.
+
+``daml.index.db.store_configuration_entry``
+-------------------------------------------
+
+A database metric. Time spent persisting a change in the ledger configuration
+provided through the configuration management service.
+
+``daml.index.db.store_ledger_entry``
+------------------------------------
+
+A database metric. Time spent persisting a transaction that has been
+successfully interpreted and is final.
+
+``daml.index.db.store_package_entry``
+-------------------------------------
+
+A database metric. Time spent storing a DAML package uploaded through
+the package management service.
+
+``daml.index.db.store_party_entry``
+-----------------------------------
+
+A database metric. Time spent storing party information as part of the
+party allocation endpoint provided by the party
+management service.
+
+``daml.index.db.store_rejection``
+---------------------------------
+
+A database metric. Time spent persisting the information that a given
+command has been rejected.
+
+``daml.lapi``
+-------------
+
+Every metrics under this namespace is a timer, one for each
+service exposed by the Ledger API, in the format:
+
+``daml.lapi.service_name.service_endpoint``
+
+As in the following example:
+
+``daml.lapi.command_service.submit_and_wait``
+
+Single call services return the time to serve the request,
+streaming services measure the time to return the first response.
+
+``jvm``
+-------
+
+Under the ``jvm`` namespace there is a collection of metrics that
+tracks important measurements about the JVM that DAML-on-SQL is
+running on, including CPU usage, memory consumption and the
+current state of threads.
+
+DAML Ledger Model Compliance
+****************************
+
+Model conformance
+=================
+
+On top of bespoke unit and integration tests, the DAML-on-SQL is
+thoroughly tested with the Ledger API Test Tool to ensure that the
+implementation conforms with the DAML Ledger Model.
+
+Performance envelope
+====================
+
+Furthermore, this implementation is regularly tested to comply
+with the DAML Ledger Implementation Performance Envelope tests
+with beta-level fidelity which ensure that DAML-on-SQL can:
+
+- process transactions as large as 1 MB
+- have a tail latency no greater than 1 second when issuing 20 pings
+- have a throughput of 20 pings per second
+
+.. note::
+
+  A "ping" is a collective name for two templates used to evaluate
+  the performance envelope. Each of the two templates, "Ping" and
+  "Pong", have a single choice allowing the controller to create
+  an instance of the complementary template, directed to the
+  original sender.
+
+Replicate performance envelope tests
+====================================
```

Comment:
```suggestion
Replicating performance envelope tests
======================================
```

Reply:
Nope, sorry, where possible I try to keep an instructional tone, it was a recommendation that [we got from a former employee](https://www.youtube.com/watch?v=8TD-20Mb_7M).

Reply:
You've used "Generating" above a lot though. 😛

Reply:
Maybe `Reproduce`?